src/main/java/Deadline.java

```diff
@@ -0,0 +1,21 @@
+public class Deadline extends Task {
+    private String dueDate;
+
+    public Deadline(String title, String dueDate) {
+        super(title);
+        this.dueDate = dueDate;
+    }
+
+    public Deadline(String title, String description, String dueDate) {
+        super(title, description);
+        this.dueDate = dueDate;
+    }
+
+    @Override
+    public void print() {
+        System.out.print("[D]");
+        super.print();
+        System.out.print(", by " + dueDate);
+        System.out.println();
+    }
```

Comment:
Is there any reason for you using `print()` instead of overriding `toString()`?

Reply:
Yes, there is, I completely forgot about toString() existing :P Whoops, I should change that. Thanks for pointing it out.